client/src/main/java/org/jboss/fuse/mvnd/client/DefaultClient.java

```diff
@@ -179,7 +179,7 @@ public ExecutionResult execute(ClientOutput output, List<String> argv) {
                         try {
                             ProcessHandle.of(di.getPid()).ifPresent(ProcessHandle::destroyForcibly);
                         } catch (Exception t) {
-                            System.out.println("Daemon " + di.getUid() + ": " + t);
+                            LOGGER.info("Daemon " + di.getUid() + ": " + t);
```

Comment:
I think we want the System.out.println() here. We prefer the message NOT to be prefixed with date, logger name, in this situation where we are killing daemons and some of those kills may fail.